2DPlatformerController/Assets/PlayerScore.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerScore : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public static int collectableCount;
+    void OnTriggerEnter2D(Collider2D target)
```

Comment:
Functions should have access modifiers for clarity (private should be fine here)